packages/widgets/src/components/Tabs.tsx

```diff
@@ -233,6 +238,19 @@ export const Tabs = withBuilder(TabsComponent, {
       helperText: 'If on, clicking an open tab closes it so no tabs are active',
       defaultValue: false,
       advanced: true
+    },
+    {
+      name: 'tabHeaderLayout',
+      type: 'enum',
+      helperText: 'Change the layout of the tab headers (uses justify-content)',
+      defaultValue: 'flex-start',
+      enum: [
+        'center',
+        'space-between',
+        'space-around',
+        'flex-start',
+        'flex-end'
```

Comment:
since some of these names won't be intuitive to people, let's use the alt format that takes a `{ label: '...', value: '...' }` so we can name then more friendly things, like

`{ label: 'Center', value: 'center' }, { label: 'Right', value: 'flex-end' }` etc